### ISSUE TITLE
[Windows] User configuration dialog for image types, etc.

### ISSUE THREAD

**Issue by GerbilSoft** [GerbilSoft/rom-properties#52]
A configuration dialog needs to be added for each UI. This configuration dialog should allow the user to configure:
* Which image type is used for systems with multiple image types. Nintendo DS, for example, has icon, cover, and box.
* Enable downloads of high-resolution images if requested by the file browser. Disable this to save bandwidth. (It may be desired to be able to use high-resolution images if they're present, but fall back to "default" if they're not.)
* Disable image downloads entirely.
* For systems that support it, use the internal icon instead of other types for small sizes. (Need to define "small": 32x32, 48x48?)

This depends on #50 for image sizes.

**Comment by DankRank** [comment]
Maybe related: #21

**Comment by GerbilSoft** [comment]
The Linux version will be postponed until the next release.
